composer/src/composer.rs

```diff
@@ -648,6 +649,14 @@ impl Builder {
         ))
     }
 
+    /// add a debug io-engine container with a name
+    pub fn add_container_dbg(self, name: &str) -> Builder {
```

Comment:
Who uses this new function ?

Reply:
Data plane.
Data plane's version Composer added a debug binary with add_container(). In this version of Composer, add_container() adds a release binary. Besides that, Build has from_dbg() method.